src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -41,6 +43,8 @@
     <Error Condition="!Exists('$(IntermediateOutputPath)')" Text="GenerateAssemblyInfo failed because IntermediateOutputPath isn't set to a valid directory" />
 
     <ItemGroup Condition="'$(MSBuildProjectExtension)' == '.csproj'">
+      <AssemblyInfoComments Include="// This code was generated by versioning.targets in the Microsoft.DotNet.BuildTools NuGet package." />
```

Comment:
I would avoid being specific about what generated this because this target gets uses in other places as well and so it will be wrong and is likely to get outdated and maintained for little benefit. I woulds simply say that it is generated by a tool an any manual changes will be overwritten.